src/sentry/mail/activity/base.py

```diff
@@ -217,14 +231,43 @@ def description_as_html(self, description, params):
 
         return mark_safe(description.format(**context))
 
-    def send(self):
+    @staticmethod
+    def get_unsubscribe_link(user_id: int, group_id: int) -> str:
+        return generate_signed_link(
+            user_id,
+            "sentry-account-email-unsubscribe-issue",
+            kwargs={"issue_id": group_id},
+        )
+
+    def update_user_context_from_group(
+        self,
+        user: Any,
+        reason: GroupSubscriptionReason,
+        context: Dict[str, Any],
+        group: Optional[Any],
+    ) -> Any:
```

Comment:
hmm, I wonder if there is any value in typing a return type as `-> Any` unless we're adding the file to the mypy config file?

Reply:
This isn't really any right? It's at least `Dict[str, Any]`, since we return the dict we pass in